tracing-subscriber/src/filter/subscriber_filters/mod.rs

```diff
@@ -99,11 +99,17 @@ pub struct FilterId(u64);
 ///
 /// [`Registry`]: crate::Registry
 /// [`Filter`]: crate::subscribe::Filter
-#[derive(Default, Copy, Clone, Eq, PartialEq)]
+#[derive(Copy, Clone, Eq, PartialEq)]
 pub(crate) struct FilterMap {
     bits: u64,
 }
 
+impl FilterMap {
+    pub(crate) const fn default() -> Self {
```

Comment:
tiny style nit, take it or leave it: i might have called this `new`; seeing uses of `FilterMap::default()` might erroneously give the impression that there's a `Default` implementation for it.  i don't think there are a whole lot of places where a `FilterMap` is constructed, so the impact on the Git diff size for changing the name of the constructor is probably not a big deal...